packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenuItemWrapper/ContextualMenuSplitButton.tsx

```diff
@@ -56,10 +59,11 @@ export class ContextualMenuSplitButton extends ContextualMenuItemWrapper {
           <div
             data-ktp-target={ keytipAttributes['data-ktp-target'] }
             ref={ (splitButton: HTMLDivElement) => this._splitButton = splitButton }
-            role={ 'button' }
-            aria-labelledby={ item.ariaLabel }
```

Comment:
why are you getting rid of aria-labelledby?

Reply:
As you mentioned, there is no way to pass it in currently.  We're not currently even using it correctly. We're just passing it the label which should be a text and not a reference to another element that should display the label.